scripts/generating_yaml.py

```diff
@@ -0,0 +1,213 @@
+                           'longitude': 149.06, # Degrees
+                          'elevation': 1000.0, # Meters
+                           'utc_offset': 10.00, # Hours
+                           'horizon': 30, # Degrees
+                           'timezone': 'Australia/Sydney'
+                          },
+             'scheduler' : {'type' : 'dispatch',
+                           'fields_file' : 'targets.yaml'},
+             'dome' : {'template_dir' : 'resources/bisque'},
+             'guider' : {'template_dir' : 'resources/bisque/guider',
+                         'image_path': '/tmp/guide_image.fits',
+                        'bin_size' : 1},
+             'mount' : {'brand': 'bisque',
+                       'model' : 45,
+                       'driver' : 'bisque',
+                       'template_dir' : 'resources/bisque'},
+             'pointing' : {'threshold' : 0.05,
+                          'exp_time' : 30,
+                          'max_iterations' : 3},
+             'state_machine' : 'huntsman',
+             'directories' : {'base' : '/var/huntsman', 'images' : 'images',
+                              'webcam' : 'webcams', 'data' : 'data',
+                              'resources' : 'POCS/resources/',
+                              'targets' : '/var/huntsman/huntsman-pocs/conf_files',
+                              'mounts' : 'POCS/resources/mounts'
+             },
+    'cameras' : {'hdr_mode' : True,
+               'auto_detect' : False,
+                'devices' : [None]}}
+
+
+# In[7]:
+
+def a(birger_serial_number):
+    return birger_serial_number
+
+def b(sbig_serial_number):
+    return sbig_serial_number
+
+def c(lens_serial_number):
+    return lens_serial_number
+
+def d(filter_ID_code):
+    return filter_ID_code
+
+def e(serial_into_USBhub_port):
+    return serial_into_USBhub_port
+
+def f(camera_into_serial_port):
+    return camera_into_serial_port
+
+def j(USBhub_SN):
+    return USBhub_SN
+
+def k(camera_into_USBhub_port):
+    return camera_into_USBhub_port
+
+
+# In[8]:
+
+birger_sn = data['birger_SN']
+birger_serial_number = interactive(a, birger_serial_number=birger_sn);
+sbig_sn = data['sbig_SN']
+sbig_serial_number = interactive(b, sbig_serial_number=sbig_sn);
+lens_sn = data['lens_SN']
+lens_serial_number = interactive(c, lens_serial_number=lens_sn);
+filter_ID = data['filter_ID']
+filter_ID_code = interactive(d, filter_ID_code=filter_ID);
+serial_into_USBhub = data['serial_into_USBhub_port']
+serial_into_USBhub_port = interactive(e, serial_into_USBhub_port=serial_into_USBhub);
+camera_into_serial = data['camera_into_serial_port']
+camera_into_serial_port = interactive(f, camera_into_serial_port=camera_into_serial);
+USBhub = data['USBhub_SN']
+USBhub_SN = interactive(j, USBhub_SN=USBhub);
+camera_into_USBhub = data['camera_into_USBhub_port']
+camera_into_USBhub_port = interactive(k, camera_into_USBhub_port=camera_into_USBhub);
+
+
+# In[9]:
+
+def h(add_device):
```

Comment:
Similar query to 'create_device'. 'add_device' doesn't seem to have a reason to be returned?